### PR TITLE
update reports with expected display

### PR DESCRIPTION
**Pull request by mkangia**
https://dimagi-dev.atlassian.net/browse/ICDS-271

Thank you @millerdev for the cool script. Modified it to use directory instead
```py
#! /usr/bin/env python3
import os
import re
import sys
from io import open

UCHAR = re.compile(r"\\u([0-9a-fA-F]{4})")

def convert_uchar(match):
    return chr(int(match.group(1), 16))

if len(sys.argv) == 1:
    print("Usage: {} FILENAMES...")

dir_path = sys.argv[1]
for file_name in os.listdir(dir_path):
    if '.json' not in file_name:
        continue
    converted = []
    filename = '/'.join([dir_path, file_name])
    print("processing {}".format(filename))
    with open(filename, encoding="utf-8") as src:
        for line in src:
            line = UCHAR.sub(convert_uchar, line)
            converted.append(line)

    with open(filename, "w", encoding="utf-8") as dst:
        dst.write("".join(converted))
```

### CODE REVIEW

File: custom/icds_reports/ucr/reports/ls_report_lbw_pre_term.json [hunk 1]
```diff
@@ -120,9 +120,9 @@
         "calculate_total": false,
         "type": "field",
         "display": {
-          "mar": "\u091c\u0928\u094d\u092e \u0924\u093e\u0930\u0940\u0916",
-          "tel": "\u0c2a\u0c41\u0c1f\u0c4d\u0c1f\u0c3f\u0c28 \u0c24\u0c47\u0c26\u0c40",
-          "hin": "\u092c\u091a\u094d\u091a\u093e",
+          "mar": "जन्म तारीख",
+          "tel": "పుట్టిన తేదీ",
+          "hin": "बच्चा",
```
Comment (mkangia) [inline custom/icds_reports/ucr/reports/ls_report_lbw_pre_term.json]: this is incorrect translation. Should be `जन्म तिथि`

File: custom/icds_reports/ucr/reports/ls_report_pregnant_women_names.json [hunk 2]
```diff
@@ -110,9 +110,9 @@
         "calculate_total": false,
         "type": "field",
         "display": {
-          "mar": "\u0938\u094d\u0924\u094d\u0930\u0940",
-          "tel": "\u0c38\u0c4d\u0c24\u0c4d\u0c30\u0c40",
-          "hin": "\u092e\u0939\u093f\u0932\u093e\u092f\u0947",
+          "mar": "स्त्री",
+          "tel": "స్త్రీ",
+          "hin": "महिलाये",
```
Comment (mkangia) [inline custom/icds_reports/ucr/reports/ls_report_pregnant_women_names.json]: This is incorrect translation. Should be `महिला`

File: custom/icds_reports/ucr/reports/ls_report_lbw_pre_term.json [hunk 3]
```diff
@@ -101,9 +101,9 @@
         "calculate_total": false,
         "type": "field",
         "display": {
-          "mar": "\u092e\u0942\u0932",
-          "tel": "\u0c2a\u0c3f\u0c32\u0c4d\u0c32\u0c32\u0c41",
-          "hin": "\u092c\u091a\u094d\u091a\u0947",
+          "mar": "मूल",
+          "tel": "పిల్లలు",
+          "hin": "बच्चे",
```
Comment (mkangia) [inline custom/icds_reports/ucr/reports/ls_report_lbw_pre_term.json]: this is incorrect translation. Should be `बच्चा`

File: custom/icds_reports/ucr/reports/ls_report_child_names.json [hunk 4]
```diff
@@ -101,9 +101,9 @@
         "calculate_total": false,
         "type": "field",
         "display": {
-          "mar": "\u092e\u0942\u0932",
-          "tel": "\u0c2a\u0c3f\u0c32\u0c4d\u0c32\u0c32\u0c41",
-          "hin": "\u092c\u091a\u094d\u091a\u093e",
+          "mar": "मूल",
```
Comment (mkangia) [inline custom/icds_reports/ucr/reports/ls_report_child_names.json]: i dont know marathi but they have used `मूल` for Child and Child name both.

File: custom/icds_reports/ucr/reports/ls_report_child_nutrition_status.json [hunk 5]
```diff
@@ -108,9 +108,9 @@
         "calculate_total": false,
         "type": "field",
         "display": {
-          "mar": "\u092e\u0942\u0932",
-          "tel": "\u0c2a\u0c3f\u0c32\u0c4d\u0c32\u0c32\u0c41",
-          "hin": "\u0928\u093e\u092e",
+          "mar": "मूल",
+          "tel": "పిల్లలు",
+          "hin": "नाम",
```
Comment (mkangia) [inline custom/icds_reports/ucr/reports/ls_report_child_nutrition_status.json]: this should be `बच्चे का नाम`

**Comment by mkangia** [comment]
@millerdev i have left comments for GS team for translations i think are incorrect. Those are not related to what this PR is addressing.

**Comment by millerdev** [comment]
> Modified it to use directory instead

Could'a done that with my original version and a little bash wildcarding 😛 

`python3 uniconvert.py dir/path/*.json`

**Review by millerdev — APPROVED** [review]

**Pull request merged**